cloudfoundry/resource_cf_app.go

```diff
@@ -537,7 +575,7 @@ func resourceAppUpdate(d *schema.ResourceData, meta interface{}) (err error) {
 	app.Buildpack = getChangedValueString("buildpack", &restage, d)
 	app.Environment = getChangedValueMap("environment", &restage, d)
 
-	if update || restart || restage {
+	if d.HasChange("name") || d.HasChange("service_binding") || d.HasChange("stopped") || d.HasChange("route") || d.HasChange("url") || d.HasChange("docker_image") || d.HasChange("git") || d.HasChange("github_release") || d.HasChange("add_content") {
```

Comment:
We usually split long line into multiple lines when too long for better reading:
e.g.:
```go
if d.HasChange("name") || 
   d.HasChange("service_binding") || 
   d.HasChange("stopped") ||
   d.HasChange("route") {
```